lib/config/migrations/custom/enabled-managers-migration.spec.ts

```diff
@@ -1,13 +1,29 @@
 import { EnabledManagersMigration } from './enabled-managers-migration';
 
 describe('config/migrations/custom/enabled-managers-migration', () => {
-  it('should replace yarn with npm', () => {
+  it('migrates', () => {
     expect(EnabledManagersMigration).toMigrate(
       {
-        enabledManagers: ['test1', 'yarn', 'test2'],
+        enabledManagers: ['test1', 'yarn', 'test2', 'regex', 'custom.regex'],
       },
       {
-        enabledManagers: ['test1', 'npm', 'test2'],
+        enabledManagers: [
+          'test1',
+          'npm',
+          'test2',
+          'custom.regex',
+          'custom.regex',
```

Comment:
Not possible/easy to deduplicate?

Reply:
I don;t we need to add code to de-duplicate since it will be a easy mistake to catch.